server/adapters/postgres.go

```diff
@@ -258,6 +258,21 @@ func (p *Postgres) Insert(eventContext *EventContext) error {
 	return nil
 }
 
+//Clean deletes all records in tableName table
+func (p *Postgres) Clean(tableName string) error {
+	wrappedTx, err := p.OpenTx()
+	if err != nil {
+		return err
+	}
+
+	if err := p.cleanTableInTransaction(wrappedTx, tableName); err != nil {
```

Comment:
here the same as in MySQL. We don't need separate transaction creation here.

Reply:
removed transaction